### PR TITLE
Add universal pwa support

### PR DESCRIPTION
**Pull request by wassertim**
please review

### CODE REVIEW

File: src/app/todo/services/todo.service.ts.orig [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import 'rxjs/add/operator/toPromise';
+import 'rxjs/add/operator/map';
+
+@Injectable()
+export class TodoService {
+
+  constructor(private http: HttpClient) { }
+
+<<<<<<< HEAD
+  list(): Observable<Array<Task>> {
```
Comment (d3ns0n) [inline src/app/todo/services/todo.service.ts.orig]: Please resolve conflicts.

Comment (d3ns0n) [inline src/app/todo/services/todo.service.ts.orig]: Please remove this file. It is a left over from resolving conflicts.

Comment (wassertim) [reply src/app/todo/services/todo.service.ts.orig]: oh god :D

**Comment by wassertim** [comment]
You start it with ./run.sh

**Comment by d3ns0n** [comment]
Now it looks good 👍

**Comment by wassertim** [comment]
I suppose it did work for you :)

**Comment by d3ns0n** [comment]
didn't try it. I am working on my branch at the moment and was too lazy to check out yours ;)

**Comment by wassertim** [comment]
ok :)

**Pull request merged**